source/core.py

```diff
@@ -1,10 +1,12 @@
-# -*- coding: UTF-8 -*-
 # A part of NonVisual Desktop Access (NVDA)
-# Copyright (C) 2006-2019 NV Access Limited, Aleksey Sadovoy, Christopher Toth, Joseph Lee, Peter Vágner,
+# Copyright (C) 2006-2021 NV Access Limited, Aleksey Sadovoy, Christopher Toth, Joseph Lee, Peter Vágner,
 # Derek Riemer, Babbage B.V., Zahari Yurukov, Łukasz Golonka
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
 
+from typing import Optional
+import wx

```

Comment:
Are you sure that this is safe? It might be intentional to ensure the order of initialization is correct (or something needs to be set up before import)

Reply:
I thought it was something worth trying to move out but perhaps something for a later day with the capacity to test it better.